common/djangoapps/student/forms.py

```diff
@@ -144,6 +148,13 @@ class AccountCreationForm(forms.Form):
             "min_length": _PASSWORD_INVALID_MSG,
         }
     )
+    password_copy = forms.CharField(
```

Comment:
@lenacom validation should happen on the front end. Microsites are moving to the new login and registration forms (cc @douglashall). The old forms are deprecated.